SPECS/rpm/rpm.spec

```diff
@@ -269,6 +276,10 @@ popd
 %{python3_sitelib}/*
 
 %changelog
+* Fri May 13 2022 Andy Caldwell <andycaldwell@microsoft.com> - 4.17.0-7
+- Add missing dependencies to rpmbuild (file, diff and patch)
+- Remove spurious `rpm` dependency on `rpm-build`
```

Comment:
Nit: line probably no longer needed after you rebased on top of your previous changes removing this dependency.

```suggestion
```